cmd/entrypoint/runner.go

```diff
@@ -8,7 +8,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
NIT: I wonder if it would be better to restore this as in this PR this might a bit confusing upfront seeing that we're changing files that does not relate with `ioutil`.

Reply:
Updated, thanks